src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/LoginPage.java

```diff
@@ -60,14 +61,16 @@ public void logUserIn(String username, String password) {
   public LoginPage get() {
     String redirectParameter = "";
 
-    try{
-      redirectParameter = URLEncoder.encode(urlBuilder.getUrlForWiki(Configuration.getWikiName()), "UTF-8");
+    try {
+      redirectParameter =
+          URLEncoder.encode(urlBuilder.getUrlForWiki(Configuration.getWikiName()), "UTF-8");
 
-    } catch (UnsupportedEncodingException e){
-      PageObjectLogging.log("encoding","problem occured during URL encoding",false);
+    } catch (UnsupportedEncodingException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)